### DISCUSSION TITLE
Tab sorted by range weight

### DISCUSSION THREAD

**Discussion by koyere** [Zrips/CMI#5376] [Feature Requests]
The suggestion is that when using the Tab function, the names in the tablist appear in order according to the weight of each range in LuckPerms, for example: that an owner appears first, then an Admin and so on.

**Comment by mrfloris** [comment]
feature requests should go to the discussion section, this is for bug reports.

**Comment by CloudeLecaw** [comment]
Jep in discussion we had this thread allready may 10 times ;)

**Comment by mrfloris** [comment]
Yay, it's a discussion now! Let's go :)

**Comment by TomLewis** [comment]
Yeah I've asked for a permission weight for tab already! So second this. Dousnt even need to be luckperns weight, can be CMI weight perm.

**Reply by CloudeLecaw** [reply]
I totally agree with you @TomLewis
It would be nice to have some kind of sorting.
Cmi permission sort of "weight" or permission systems weight would be wonderfull to have this.

Site fact:
Lets take a look here..
So many other times ppl now wished some type of sorting

https://github.com/Zrips/CMI/discussions/931
https://github.com/Zrips/CMI/discussions/5233
https://github.com/Zrips/CMI/discussions/3571
https://github.com/Zrips/CMI/discussions/3912
https://github.com/Zrips/CMI/discussions/6031
https://github.com/Zrips/CMI/discussions/4695
https://github.com/Zrips/CMI/discussions/1480
https://github.com/Zrips/CMI/discussions/653
https://github.com/Zrips/CMI/discussions/3479
https://github.com/Zrips/CMI/discussions/2671
https://github.com/Zrips/CMI/discussions/1492

It really would be nice if there would be a reaction or adding a feature like that.

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending and descending per type, and types vary from permission node weight, to baltop rank, etc.
